all-requirements.txt

```diff
@@ -10,3 +10,6 @@ tornado==4.2
 
 # For simpl.middleware.cors
 WebOb==1.4.1
+
+# MongoDB
+pymongo==2.8.1
```

Comment:
Preferably, we'd be able to use pymongo > 3.0. It seems like this would require some amount of work, but it might be hard to estimate how much. 

https://api.mongodb.org/python/current/changelog.html#changes-in-version-3-0